src/overlays/actors/ovl_En_Po_Sisters/z_en_po_sisters.c

```diff
@@ -146,15 +146,15 @@ static InitChainEntry sInitChain[] = {
 
 // clang-format off
 // PoSisters have their own flags variable for cross function behavior detection
-#define POSISTERS_FLAG_CLEAR                 (0)
-#define POSISTERS_FLAG_CHECK_AC              (1 << 0)
-#define POSISTERS_FLAG_UPDATE_SHAPE_ROT      (1 << 1)
-#define POSISTERS_FLAG_CHECK_Z_TARGET        (1 << 2) // meg doesnt go invis if you ztarget her for too long
-#define POSISTERS_FLAG_MATCH_PLAYER_HEIGHT   (1 << 3) // the po is attempting to level with player's height
-#define POSISTERS_FLAG_UPDATE_BGCHECK_INFO   (1 << 4)
-#define POSISTERS_FLAG_UPDATE_FIRES          (1 << 5) // firePos updated to match limb in PostLimbDraw
-#define POSISTERS_FLAG_REAL_MEG_ROTATION     (1 << 6) // real meg rotates different than her clones for one cycle
-#define POSISTERS_FLAG_DRAW_TORCH            (1 << 7)
+#define POE_SISTERS_FLAG_CLEAR                 (0)
+#define POE_SISTERS_FLAG_CHECK_AC              (1 << 0)
+#define POE_SISTERS_FLAG_UPDATE_SHAPE_ROT      (1 << 1)
+#define POE_SISTERS_FLAG_CHECK_Z_TARGET        (1 << 2) // meg doesnt go invis if you ztarget her for too long
+#define POE_SISTERS_FLAG_MATCH_PLAYER_HEIGHT   (1 << 3) // the po is attempting to level with player's height
```

Comment:
```suggestion
#define POE_SISTERS_FLAG_CHECK_Z_TARGET        (1 << 2) // Meg doesnt go invisible if you ztarget her for too long
#define POE_SISTERS_FLAG_MATCH_PLAYER_HEIGHT   (1 << 3) // the Poe is attempting to level with player's height
```
And similar capitalisation elsewhere.